src/tests/Myr.test.js

```diff
@@ -1,4 +1,5 @@
 import Myr from "../myr/Myr";
```

Comment:
Please add tests for your newly added MYR APIs

Reply:
(mentioned above) this should include getter functions